promotions/lib/controllers/backend/solidus_promotions/admin/benefits_controller.rb

```diff
@@ -28,7 +28,7 @@ def edit
         if params.dig(:benefit, :calculator_type)
           @benefit.calculator_type = params[:benefit][:calculator_type]
         end
-        render layout: false
+        render layout: false, status: :unprocessable_entity
```

Comment:
This should not have the `:status_unprocessable_entity`.  This is the initial render of the layout for edit.

Reply:
Thanks :) https://github.com/solidusio/solidus/pull/5997/files